### DISCUSSION TITLE
Merge all pacman mirror lists into one

### DISCUSSION THREAD

**Discussion by wszqkzqk** [msys2/MSYS2-packages#3267] [Ideas]
It's not convient to edit lists of both msys and mingw separately, we may merge them into one file, and use `$var` to distinct.

**Comment by lazka** [comment]
Pacman only supports `$repo` and `$arch`, see https://archlinux.org/pacman/pacman.conf.5.html#_repository_sections

**Comment by asmwarrior** [comment]
Hi, I see recently, some mirror config files are not needed. When I looked at the git history, I see this:

[pacman-mirrors: remove env specific mirror lists (#5886) · msys2/MSYS2-packages@b877f04](https://github.com/msys2/MSYS2-packages/commit/b877f040f59ff6fee22610bc5dc68fce1626821f)

This way, only 2 files were used:

See here: [Package Mirrors - MSYS2](https://www.msys2.org/docs/mirrors/)

Can you add some news about this, maybe some users like me will get confused about this. As I see there are many unused files in my `/etc/pacman.d` folder.

Thanks.

**Reply by lazka** [reply]
The files are not used for 4+ years now, and the referenced commit just removed them. In case you changed them they will not be removed but renamed to end in ".pacsave".

So for 99% of users those files no longer exist, so I don't think it's worth a news entry.

**Reply by asmwarrior** [reply]
OK, I see, thanks for the explanation.
